gradle/spotbugs-exclude.xml

```diff
@@ -28,8 +28,19 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <!-- Disable warnings about mutable objects and the use of public fields.
             EI_EXPOSE_REP: May expose internal representation by returning reference to mutable object
             EI_EXPOSE_REP2: May expose internal representation by incorporating reference to mutable object
+            MS_EXPOSE_REP: Public static method may expose internal representation by returning array
             MS_PKGPROTECT: Field should be package protected -->
-        <Bug pattern="EI_EXPOSE_REP,EI_EXPOSE_REP2,MS_PKGPROTECT"/>
+        <Or>
+            <Bug pattern="EI_EXPOSE_REP"/>
+            <Bug pattern="EI_EXPOSE_REP2"/>
+            <Bug pattern="MS_EXPOSE_REP"/>
+            <Bug pattern="MS_PKGPROTECT"/>
+        </Or>
+    </Match>
+    <Match>
+        <!-- Disable warnings about Random object.
+            DMI_RANDOM_USED_ONLY_ONCE: Random object created and used only once -->
+        <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE"/>
```

Comment:
Are we disabling this one due to false positives?

Reply:
Yes.

1. Random object created and used only once in new `kafka.tools.ConsoleConsumer$ConsumerConfig(String[])`
2. Random object created and used only once in new `kafka.tools.ConsumerPerformance$ConsumerPerfConfig(String[])`